runtime/onert/core/include/exec/Execution.h

```diff
@@ -243,11 +243,8 @@ class Execution
   void sholudStop();
 
 private:
-  const std::unique_ptr<IExecutor> &primary_executor() const
-  {
-    return _executors->at(ir::SubgraphIndex{0});
-  };
-  std::unique_ptr<IExecutor> &primary_executor() { return _executors->at(ir::SubgraphIndex{0}); };
+  const IExecutor *primary_executor() const { return _executors->primary_executor(); }
+  IExecutor *primary_executor() { return _executors->primary_executor(); }
```

Comment:
I found `primary_executor()` is used to get input/output size. But by multi model, primary executor's input/output is not always same with package input/output. We need to fix this. (nnfwapi and may return invalid input/output size on multi model)